src/__tests__/commentCounter.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import commentsCounter from '../modules/commentCount.js';
+
+describe('Counters', () => {
+  it('Comments counters', () => {
+    document.body.innerHTML = `<div class="left">
+  <p class="eachScore">
+    2022-09-09
+    <span>Nganje:</span>
+  </p>
+  <span>
+    <p class="numberSc">Great</p>
+  </span>
+</div>`;
+    const count = document.querySelectorAll('.left');
+    expect(commentsCounter(count)).toBe(0);
+  });
+});
```

Comment:
- [ ] All the tests fail after running the command `npm test` in the terminal. Please try to handle that. If the user needs to install some packages in order to run the tests, it will be preferable to include that in your README file.

<img width="407" alt="image" src="https://user-images.githubusercontent.com/77190203/189487178-5967e9c5-c11b-4c4b-9c74-457f8df020ad.png">